lib/rules.rb

```diff
@@ -1,20 +0,0 @@
-class Rules
```

Comment:
What was the reason for moving this logic into the game class?

Reply:
@pelensky I did have my logic in a 'Rules` class but I ended up having 3 if statements: one in `Rules` determining the shape that won, and two in game - which player had won and presenting the winning message. This way cut it down to two if statements. I could move that logic back to a `Rules` class - what do you think?